### PR TITLE
TN-2770 /api/user/<id>/research_study API

### PR DESCRIPTION
**Pull request by pbugni**

### CODE REVIEW

File: portal/views/research_study.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+@crossdomain()
+@oauth.require_oauth()
+def rs_for_user(user_id):
+    """Returns simple JSON for research study user is associated with
+
+    ---
+    tags:
+      - User
+      - ResearchStudy
+    operationId: rs_for_user
```
Comment (ivan-c) [inline portal/views/research_study.py:28]: I think this appears in in title for an API in SwaggerUI and may not be the most obvious
```suggestion
    operationId: research_studies_for_user
```

**Review by ivan-c — APPROVED** [review]
Looks good! minor nitpick on naming but ready to merge

**Review by achen2401 — APPROVED** [review]
Thank you, Paul

File: portal/views/research_study.py (hunk 1 above)
Comment (pbugni) [reply portal/views/research_study.py:28]: strange - this didn't show in the github ui till after i hit merge.  tempting to change in develop being a comment and all :)

**Pull request merged**